webapi/src/main/scala/org/knora/webapi/responders/v1/ResourcesResponderV1.scala

```diff
@@ -1176,66 +1176,6 @@ class ResourcesResponderV1 extends Responder {
 
 
     /**
-      * Implements a pre-update check to ensure that an [[UpdateValueV1]] has the correct type for the `knora-base:objectClassConstraint` of
```

Comment:
I just had a quick look and it just looks like you took everything unchanged from `develop`. What changes did you make in `ResourcesResponderV1`?